guides/v2.1/cloud/release-notes/cloud-tools.md

```diff
@@ -18,14 +18,84 @@ You can list the available ece-tools commands using:
+
+- JIRA--MAGECLOUD-2363-->**Apply custom changes to your {{site.data.var.ece}} application during the build phase**—We split the build phase into two separate processes so that you can apply custom changes to the generated static content before packaging the application for deployment. The *build:generate* process generates code, applies patches, and generates static content. The *build:transfer* phase deploys the application. See [Application hooks]({{ page.baseurl }}//cloud/project/project-conf-files_magento-app.html#hooks).
+
+-  JIRA--MAGECLOUD-2445--> **Cron scheduling improvements**—Improved the cron job management process during the deploy phase to prevent database locks and other critical issues. Now, all cron jobs stop during the deploy phase and restart after the deploy phase completes.
+
+-  **Environment configuration checks**—Improved validation of the environment configuration to proactively warn customers about version incompatibilities and configuration errors before building and deploying {{site.data.var.ece}}.
+
+   -  JIRA--MAGECLOUD-2183-->Added version-specific validation to identify unsupported or deprecated {{site.data.var.ece}} environment variables
+   and values.
+
+   -  JIRA--MAGECLOUD-2389-->Added an Elasticsearch compatibility check. Now, the deployment fails if the `.magento/services.yaml` file specifies an Elasticsearch version that is incompatible with the elasticsearch composer package used to install {{site.data.var.ee}} v2.1.4 and later. Previously, you could deploy with an unsupported version of the Elasticsearch service, which caused product catalog issues after site deployment. See [Setup Elasticsearch]({{ page.baseurl }}/cloud/project/project-conf-files_services-elastic.html).
+
+   -  JIRA--MAGECLOUD-2156-->Improved validation of environment variables to identify configuration settings that can cause conflicts during the {{site.data.var.ece}} build, deploy, and post-deploy phases. For example, a warning message displays during the install and upgrade process if the global setting for static site content deployment conflicts with settings on the build or deploy phase.
+
+-  **Environment variable updates**—Changed the following environment variables:
+
+    -  JIRA--MAGECLOUD-2435-->**[SKIP_HTML_MINIFICATION global variable]({{ page.baseurl }}/cloud/env/variables-intro.html##skip_html_minification)**—Changed the default value from `false` to `true` to enable the on-demand static content minification setting in the default environment configuration. This change decreases downtime significantly and is required for zero-downtime deployments. See [Global variables]({{ page.baseurl }}/cloud/env/variables-intro.html##skip_html_minification).
```

Comment:
static content minification -> HTML content minification
We need to be clear here, what is generated on demand.

Reply:
Both links go to the same place. We only need one.